ui/app/serializers/clients/config.js

```diff
@@ -5,11 +5,17 @@
 
 import ApplicationSerializer from '../application';
 
-export default ApplicationSerializer.extend({
+export default class ClientsConfigSerializer extends ApplicationSerializer {
+  // these attrs are readOnly
+  attrs = {
+    billingStartTimestamp: { serialize: false },
+    minimumRetentionMonths: { serialize: false },
+    reportingEnabled: { serialize: false },
+  };
+
   normalizeResponse(store, primaryModelClass, payload, id, requestType) {
     if (!payload.data) {
-      // CBS TODO: Remove this if block once API is published
-      return this._super(store, primaryModelClass, payload, id, requestType);
+      return super.normalizeResponse(store, primaryModelClass, payload, id, requestType);
```

Comment:
could this also be the following? no need to change, just for my personal knowledge
```suggestion
      return super.normalizeResponse(...arguments);
```